string topicName
string newValue
---
string setAnswer
